helm/generic-node/templates/generic-node.yaml

```diff
@@ -0,0 +1,292 @@
+              - matchExpressions:
+                - key: "cloud.google.com/gke-preemptible"
+                  operator: In
+                  values: ["true"]
+      initContainers:
+      {{- if .Values.libp2p.staticKeypair.enabled}}
+      - name: libp2p-init-my-service
+        image: busybox:latest
+        command: ['sh', '-c', '/bin/cp /libp2p-keys/* /root/libp2p-keys && /bin/chmod -R 0700 /root/libp2p-keys/']
+        volumeMounts:
+        - mountPath: /libp2p-keys
+          name: source-libp2p-keys
+        - mountPath: /root/libp2p-keys
+          name: destination-libp2p
+      {{- end }}
+      {{- if .Values.features.privateKeySecret.enabled}}
+      - name: fix-perms
+        image: busybox:latest
+        command: [ 'sh', '-c', 'for dir in keys echo-keys faucet-keys; do [ -d /$dir ] && /bin/cp /$dir/* /wallet-keys; done; /bin/chmod 0700 /wallet-keys']
+        volumeMounts:
+        - mountPath: "/keys/"
+          name: private-keys
+          readOnly: true
+        - mountPath: /wallet-keys
+          name: wallet-keys
+      {{- end }}
+      containers:
+      - name: daemon
+        image: "{{.Values.image.repository }}:{{.Values.image.tag}}"
+        imagePullPolicy: {{.Values.image.pullPolicy}}
+        {{- if .Values.features.customEntrypoint.enabled}}
+        command: [ {{ .Values.features.customEntrypoint.command}} ]
+        {{- end }}
+        volumeMounts:
+        - name: destination-libp2p
+          mountPath: /root/libp2p-keys
+          readOnly: true
+        - name: config-dir
+          mountPath: "/root/.mina-config"
+        {{- if .Values.features.uploadBlocksToGCloud.enabled}}
+        - name: gcloud-keyfile
+          mountPath: "/gcloud/"
+        {{- end }}
+        {{- if .Values.features.runtimeConfig.enabled}}
+        - name: daemon-config
+          mountPath: "/config/"
+        {{- end }}
+        {{- if .Values.libp2p.staticKeypair.enabled}}
+        - name: destination-libp2p
+          mountPath: "/root/libp2p-keys"
+        {{- end }}
+        {{-  if .Values.features.persistence.enabled}}
+        - name: {{.Values.features.persistence.name}}
+          mountPath: "{{.Values.features.persistence.mountPath}}"
+        {{- end }}
+        resources:
+          limits:
+          requests:
+            memory: {{.Values.resources.requests.memory}}
+            cpu: {{.Values.resources.requests.cpu}}
+        args:
+        - "mina-daemon"
+        - "run"
+        - "daemon"
+        - "--log-level"
+        - "Debug"
+        - "--log-json"
+        - "--client-port"
+        - "$(DAEMON_CLIENT_PORT)"
+        - "--rest-port"
+        - "$(DAEMON_REST_PORT)"
+        - "--external-port"
+        - "$(DAEMON_EXTERNAL_PORT)"
+        - "--metrics-port"
+        - "$(DAEMON_METRICS_PORT)"
+        - "-run-snark-coordinator"
+        - "$(CODA_SNARK_KEY)"
+        - "-snark-worker-fee"
+        - "$(CODA_SNARK_FEE)"
+        - "-work-selection"
+        - "$(WORK_SELECTION)"
+        - "-u"
```

Comment:
-u /code/watchdog.py will not work at all

Reply:
Same with --incluster true, --namespace  "", these are all flags to the watchdog python script, not flags to the daemon